expression/constant_propagation.go

```diff
@@ -158,6 +146,51 @@ func (s *propagateConstantSolver) validPropagateCond(cond Expression, funNameMap
 	return nil, nil
 }
 
+// tryToReplaceCond aims to replace all occurances of column 'src' and try to replace it with 'tgt' in 'cond'
```

Comment:
We need a more detailed comment here,
an example may help.

Reply:
Yeah,I will try to state more details